agile_problem2.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Problem2 do
+	def largest_2_numbers(), do: -2
+	
+	def largest_2_numbers(list) do
+		#First check that we have at least 2 elements. Otherwise it's useless
+		if Enum.count(list)<2 do
+			-1

```

Comment:
In elixir your will use [atoms](https://elixir-lang.org/getting-started/basic-types.html#atoms) to represent this, they have the benefit of being more readable and of course constants.